src/e2e/resources/data/InstructorNotificationsPageE2ETest.json

```diff
@@ -0,0 +1,108 @@
+{
+  "accounts": {
+    "INotifications.instr": {
+      "googleId": "tm.e2e.INotifications.instr",
+      "name": "Teammates Test",
+      "email": "INotifications.instr@gmail.tmt",
+      "readNotifications": {
+        "notification4": "2099-04-04T00:00:00Z"
+      }
+    }
+  },
+  "courses": {
+    "typicalCourse1": {
+      "id": "tm.e2e.INotifications.course1",
+      "name": "Typical Course 1",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "Africa/Johannesburg"
+    }
+  },
+  "instructors": {
+    "INotifications.instr": {
+      "googleId": "tm.e2e.INotifications.instr",
+      "courseId": "tm.e2e.INotifications.course1",
+      "name": "Teammates Instr1",
+      "email": "INotifications.instr@gmail.tmt",
+      "isArchived": false,
+      "role": "Co-owner",
+      "isDisplayedToStudents": true,
+      "displayedName": "Instructor",
+      "privileges": {
+        "courseLevel": {
+          "canModifyCourse": true,
+          "canModifyInstructor": true,
+          "canModifySession": true,
+          "canModifyStudent": true,
+          "canViewStudentInSections": true,
+          "canViewSessionInSections": true,
+          "canSubmitSessionInSections": true,
+          "canModifySessionCommentsInSections": true
+        },
+        "sectionLevel": {},
+        "sessionLevel": {}
+      }
+    }
+  },
+  "notifications": {
+    "notification1": {
+      "notificationId": "notification1",
+      "startTime": "2011-01-01T00:00:00Z",
+      "endTime": "2099-01-01T00:00:00Z",
+      "createdAt": "2011-01-01T00:00:00Z",
+      "updatedAt": "2011-01-01T00:00:00Z",
```

Comment:
`updatedAt` might not be necessary for test data now, considering it is transient in NotificationAttribute now. This also applies to the typical data bundle.